### ISSUE TITLE
it_test_authorization.py:9-15: After #298 is finished...

### ISSUE THREAD

**Issue by 0pdd** [timelesslounge/timelessis#358]
The puzzle `329-4c9067b6` from #329 has to be resolved: 

https://github.com/timelesslounge/timelessis/blob/654715c9686ecf0c628021f586d54381a67388eb/tests/integration/it_test_authorization.py#L9-L15

The puzzle was created by Krzysztof Krason on 20-Feb-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#358](https://github.com/timelesslounge/timelessis/issues/358) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#358](https://github.com/timelesslounge/timelessis/issues/358) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/f1f6ee72-ce75-463b-ad30-4da3cad82006); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/timelesslounge/timelessis/issues/358#issuecomment-469528101))

@paulodamaso The user @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) resigned from [#358](https://github.com/timelesslounge/timelessis/issues/358), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
The job [#358](https://github.com/timelesslounge/timelessis/issues/358) assigned to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/e6f59512-9a54-41f7-b22e-ce096a1020d0); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by 0crat** [comment]
@vladarefiev[/z](https://www.0crat.com/u/vladarefiev) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
The puzzle `329-4c9067b6` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#526](https://github.com/timelesslounge/timelessis/issues/526) is still not solved.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vladarefiev[/z](https://www.0crat.com/u/vladarefiev), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#358](https://github.com/timelesslounge/timelessis/issues/358) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
> @0crat quality good ([here](https://github.com/timelesslounge/timelessis/issues/358#issuecomment-471981691))

@ypshenychka There is no quality review for [#358](https://github.com/timelesslounge/timelessis/issues/358), no performer

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @vladarefiev[/z](https://www.0crat.com/u/vladarefiev)

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@0pdd the puzzle [#unknown]() is still not solved.
